.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: python
+dist: xenial
 python:
-  - "2.7"
+  - "3.7"
```

Comment:
Should this list both 2.7 and 3.7?

Reply:
No. This is saying what the base machine on travis is running (which needs to be python3.7 if we want to run tests in pyhton3.7). If we have both, we'll run six tests (two of which will fail).